traffic_router/connector/src/main/java/com/comcast/cdn/traffic_control/traffic_router/protocol/RouterNioEndpoint.java

```diff
@@ -69,23 +68,11 @@ synchronized private void replaceSSLHosts(final Map<String, HandshakeData> sslHo
                 lastHostName = sslHostConfig.getHostName();
             }
 
-            if (firstAlias && ! "".equals(alias)) {
+            if (CertificateRegistry.DEFAULT_SSL_KEY.equals(alias)) {
                 // One of the configs must be set as the default
                 setDefaultSSLHostConfigName(sslHostConfig.getHostName());
-                firstAlias = false;
             }
         }
-
-    }
-
-    synchronized public void reloadSSLHosts(final Map<String, HandshakeData> cr) {
-        replaceSSLHosts(cr);
-
-        for (final HandshakeData data : cr.values()) {
```

Comment:
So this is unnecessary now since TR just always sets the default cert and will never swap in a "default" cert from the new sslkeys.json?

Reply:
It was never necessary. All of those same steps happen in the `replaceSSLHosts` method and are for some reason repeated again. I just changed the `replaceSSLHosts` to be public and changed the callback to call it.